<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-22T17:49:20 IST" name="Default suite" finished-at="2022-09-22T17:49:32 IST" duration-ms="12701">
    <groups>
    </groups>
    <test started-at="2022-09-22T17:49:20 IST" name="Default test" finished-at="2022-09-22T17:49:32 IST" duration-ms="12701">
      <class name="testNG.TestNG8">
        <test-method is-config="true" signature="setEnv()[pri:0, instance:testNG.TestNG8@4bb33f74]" started-at="2022-09-22T17:49:20 IST" name="setEnv" finished-at="2022-09-22T17:49:32 IST" duration-ms="12555" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setEnv -->
        <test-method signature="enterCred(java.lang.String,java.lang.String)[pri:0, instance:testNG.TestNG8@4bb33f74]" started-at="2022-09-22T17:49:32 IST" name="enterCred" finished-at="2022-09-22T17:49:32 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void testNG.TestNG8.enterCred(java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : dataset]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void testNG.TestNG8.enterCred(java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : dataset
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterCred -->
      </class> <!-- testNG.TestNG8 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
